### ISSUE TITLE
Wallpaper parts jump around

### ISSUE THREAD

**Issue by FelixSchuetze** [patzly/doodle-android#93]
I'm using doodle on my Pixel 6 with Android 14 beta. Sadly, when the tilt effect is enabled, parts of the wallpaper will start jumping randomly every few seconds and change their size.

**Comment by patzly** [comment]
That's weird, especially if it is only on 14.. Can you please upload a screen recording?

**Comment by FelixSchuetze** [comment]
Link to Google photos:
[https://photos.app.goo.gl/cN3L1AUdX5Zk6equ7](https://photos.app.goo.gl/cN3L1AUdX5Zk6equ7)

**Comment by FelixSchuetze** [comment]
I'm not sure how to upload videos on github

**Comment by FelixSchuetze** [comment]
Accelerometer and Gyroscope seem to work fine, just tested them. Also, the slow and well animated movements of the wallpaper happen when I move the phone. Only the stutters aren't intended behavior.

**Comment by TheAlienDrew** [comment]
I mean, since it's only the beta that this is happening on, maybe it's something that they're (Google) working on already? I wouldn't try programming app features to the beta, until the update is released as stable. Otherwise, you might just happen to program something useless by the time it comes out.

**Comment by patzly** [comment]
Yeah I probably have to wait until I have Android 14 on my Pixel, I only use the stable channel and trying to fix an issue I cannot reproduce is very hard. Hopefully I can find a fix for that soon!
